Ada_Zangemann-cs.txt

```diff
@@ -5,197 +5,138 @@
 ## 0 ##
 
 [[IDX:
-toc:Table of Contents
+toc: Obsah
```

Comment:
```suggestion
toc:Obsah
```
Tady ještě ujela mezera, nevím jestli by to pak vadilo (při renderu), ale aspoň v rámci konzistence...